pageserver/src/bin/pageserver.rs

```diff
@@ -226,6 +228,19 @@ fn start_pageserver(
     launch_ts: &'static LaunchTimestamp,
     conf: &'static PageServerConf,
 ) -> anyhow::Result<()> {
+    // Monotonic time for later calculating startup duration
+    let started_startup_at = Instant::now();
+
+    let startup_checkpoint = move |phase: &str| {
+        STARTUP_DURATION.with_label_values(&[phase]).set(
+            started_startup_at
+                .elapsed()
+                .as_millis()
+                .try_into()
+                .unwrap_or(u64::MAX),
```

Comment:
With metrics (prometheus) it might be more natural to expose this as f64 seconds. Could you restructure this so that it would return the Duration as well, so we wouldn't log one and have metrics report the other?

I think I will still keep the `[iI]nitial ` in my log queries whenever looking at re-deployments, as the metrics are scraped at every 60s.